src/event/player/PlayerResourcePackOfferEvent.php

```diff
@@ -38,6 +47,10 @@ public function __construct(
 		private bool $resourcePacksRequired
 	){}
 
+	/**
+	 * Returns an object containing self-proclaimed information about the connecting player. The player has been
+	 * authenticated at this point, but the information should still be treated with caution.
```

Comment:
This is unnecessarily over-descriptive, and the caution is unnecessary - once they are authenticated the information can be fully trusted